{"name": "Test login attempt with wrong credentials", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'", "trace": "self = <src.tests.test_selenium.TestLogin object at 0x000001FA23B791D0>\nlogin_data = ('Alexander_Artemov', 'test')\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ae01cda881b0d3c182b60bf45e4ab9ae\")>\n\n    @allure.step\n    @allure.tag('UI')\n    @allure.feature('User management')\n    @allure.story('Login')\n    @allure.title(\"Test login attempt with wrong credentials\")\n    @allure.severity(allure.severity_level.NORMAL)\n    @pytest.mark.parametrize(\"login_data\", [(\"alex_art\", \"test\"),\n                                            (\"alex_art\", \"\"),\n                                            (\"\", \"\"),\n                                            (\"Alexander_Artemov\", \"test\")])\n    def test_login_incorrect(self, login_data, driver):\n        login_page = LoginPage(driver)\n        login_page.go(\"http://jira.hillel.it:8080/\")\n        login_page.login(*login_data)\n    \n>       allure.attach('screenshot', self.driver.get_screenshot_as_png(), type=AttachmentType.PNG)\nE       AttributeError: 'TestLogin' object has no attribute 'driver'\n\nsrc\\tests\\test_selenium.py:61: AttributeError"}, "steps": [{"name": "test_login_incorrect", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'\n", "trace": "  File \"c:\\users\\aartemov\\pycharmprojects\\pythoncourse\\phase4\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 141, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\aartemov\\PycharmProjects\\pythoncourse\\phase5\\src\\tests\\test_selenium.py\", line 61, in test_login_incorrect\n    allure.attach('screenshot', self.driver.get_screenshot_as_png(), type=AttachmentType.PNG)\n"}, "parameters": [{"name": "login_data", "value": "('Alexander_Artemov', 'test')"}, {"name": "driver", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"ae01cda881b0d3c182b60bf45e4ab9ae\")>"}], "start": 1537454096136, "stop": 1537454110936}], "parameters": [{"name": "login_data", "value": "('Alexander_Artemov', 'test')"}], "start": 1537454096135, "stop": 1537454110936, "uuid": "1ae74947-120d-4f51-ac31-15c6173b6d9b", "historyId": "384da9776e08e2dbe65b4e7101c5fe29", "fullName": "src.tests.test_selenium.TestLogin#test_login_incorrect[login_data3]", "labels": [{"name": "tag", "value": "UI"}, {"name": "severity", "value": "normal"}, {"name": "story", "value": "Login"}, {"name": "feature", "value": "User management"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_selenium"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "odp1210013"}, {"name": "thread", "value": "22624-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_selenium"}]}